src/components/MarketDetail/index.js

```diff
@@ -294,6 +297,19 @@ class MarketDetail extends Component {
     )
   }
 
+  renderMarketGraph() {
+    const { market, marketGraph } = this.props
+    if (!marketGraph.length) {
```

Comment:
What about:
```
const hasData = marketGraph.length;
....

<div className="container">
    { hadData
        ? <MarketGraph data={marketGraph} market={market} />
        :  <LoadingIndicator className="marketGraph__spinner" />
    }
</div>
```

Reply:
@apanizo VM119:1 Uncaught ReferenceError: hadData is not defined 😆